internal/provisioner/helm/README.md

```diff
@@ -0,0 +1,272 @@
+# Plain Provisioner
+
+## Summary
+
+The `helm` provisioner is able to instantiate a given `helm+v0` bundle with a specified helm chart onto a cluster and then install the helm chart,
```

Comment:
This brings up the interesting point of whether we need the `+v0` -- since helm is an external format, I'm not sure we are going to have the same metadata questions we have regarding our own, internal, `plain` format. I'm personally for dropping the v0 and just calling them helm bundles in the short term. 

This would create less confusion on what helm bundles are and how to use them.

Reply:
OK.  That makes sense.

Reply:
I'd probably call them helm v3 bundles. Helm made some minor (maybe breaking, can't remember?) changes between v2 and v3.

Reply:
"v3" makes sense.  I'll put "+v3".